crates/nu-utils/standard_library/std.nu

```diff
@@ -296,3 +296,721 @@ export def "log debug" [message: string] {
 
     print --stderr $"(ansi default_dimmed)DEBUG ($message)(ansi reset)"
 }
+
+def error-fmt [] {
+    $"(ansi red)($in)(ansi reset)"
+}
+
+def throw-error [error: string, msg: string, span: record] {
+    error make {
+        msg: ($error | error-fmt)
+        label: {
+            text: $msg
+            start: $span.start
+            end: $span.end
+        }
+    }
+}
+
+def module-not-found-error [span: record] {
+    throw-error "std::help::module_not_found" "module not found" $span
+}
+
+def alias-not-found-error [span: record] {
+    throw-error "std::help::alias_not_found" "alias not found" $span
+}
+
+def extern-not-found-error [span: record] {
+    throw-error "std::help::extern_not_found" "extern not found" $span
+}
+
+def operator-not-found-error [span: record] {
+    throw-error "std::help::operator_not_found" "operator not found" $span
+}
+
+def command-not-found-error [span: record] {
+    throw-error "std::help::command_not_found" "command not found" $span
+}
+
+def print-help-header [
+    text: string
+    --no-newline (-n): bool
+] {
+    let header = $"(ansi green)($text)(ansi reset):"
+
+    if $no_newline {
+        print -n $header
+    } else {
+        print $header
+    }
+}
+
+def show-module [module: record] {
+    if not ($module.usage? | is-empty) {
+        print $module.usage
+        print ""
+    }
+
+    print-help-header -n "Module"
+    print $" ($module.name)"
+    print ""
+
+    if not ($module.commands? | is-empty) {
+        print-help-header "Exported commands"
+        print -n "    "
+
+        let commands_string = (
+            $module.commands
+            | each {|command|
+                $"($command) (char lparen)(ansi cyan_bold)($module.name) ($command)(ansi reset)(char rparen)"
```

Comment:
This one is not correct. I think it's better to drop the cyan version for now because I don't think there is a way to query it correctly from inside Nushell.

The bold cyan shows the actual name of the exported command inside the current scope which depends on how you import it. `use std` will have `std` prefix while `use std *` won't. Later, we might support renaming of modules which would lead to a completely different prefix. We'd probably need to expose the ID of commands inside `$nu` in order to be able to link the module's command with an existing symbol in the current scope.

Reply:
should be good in 3c54ca6b3dced82bbb3641c7803e7a4197045fc3